### PR TITLE
Fix: handle empty LOC_CHASSIS_TABLE

### PR DESCRIPTION
**Pull request by qiluo-msft**
**- What I did**
We notice sometimes an error message. This PR will handle the case when the specific Redis table is not available/empty.
```
Jan  1 23:11:19.144294 vlab-01 ERR snmp#snmp-subagent [ax_interface] ERROR: MIBUpdater.start() caught an unexpected exception during update_data()
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/ax_interface/mib.py", line 37, in start
    self.reinit_data()
  File "/usr/local/lib/python3.7/dist-packages/sonic_ax_impl/mibs/ieee802_1ab.py", line 115, in reinit_data
    self.loc_chassis_data['lldp_loc_sys_cap_supported'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_supported'])
KeyError: 'lldp_loc_sys_cap_supported'
```

**- How I did it**

**- How to verify it**
Unit test

**- Description for the changelog**

### CODE REVIEW

**Review by lguohan — APPROVED** [review]

File: src/sonic_ax_impl/mibs/ieee802_1ab.py [hunk 1]
```diff
@@ -112,8 +112,10 @@ def reinit_data(self):
         # establish connection to application database.
         Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB)
         self.loc_chassis_data = Namespace.dbs_get_all(self.db_conn, mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE)
-        self.loc_chassis_data['lldp_loc_sys_cap_supported'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_supported'])
-        self.loc_chassis_data['lldp_loc_sys_cap_enabled'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_enabled'])
+        if self.loc_chassis_data:
+            self.loc_chassis_data['lldp_loc_sys_cap_supported'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_supported'])
+            self.loc_chassis_data['lldp_loc_sys_cap_enabled'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_enabled'])
```
Comment (lguohan) [inline src/sonic_ax_impl/mibs/ieee802_1ab.py:117]: where is the unit test?

Comment (qiluo-msft) [reply src/sonic_ax_impl/mibs/ieee802_1ab.py:117]: Existing test case `test_getnextpdu_lldpLocSysCapSupported`

Comment (lguohan) [reply src/sonic_ax_impl/mibs/ieee802_1ab.py:117]: the test also passes without this fix?

Comment (qiluo-msft) [reply src/sonic_ax_impl/mibs/ieee802_1ab.py:117]: Yes. This is a corner case on DUT with low possibility to repro.
The old impact is syslog error message, and the snmpagent recovered soon after the Redis table is ready. Not a critical bug.
